src/main/java/seedu/duke/common/ErrorMessages.java

```diff
@@ -18,6 +18,7 @@ public enum ErrorMessages {
     ERROR_GLOBAL_INVALID_PERIOD("Type of period given is invalid, please check your input!"),
     ERROR_GLOBAL_INVALID_NUMBER("Number for period stats given is invalid, please check your input!"),
     ERROR_GLOBAL_NUMBER_NOT_NUMERIC("Non-integer detected, please enter a numerical integer!"),
+    ERROR_GLOBAL_NON_NUMERIC_INDEX("Non-numerical index detected, please enter a numerical index!"),
```

Comment:
Changes look good, there's a clear distinction between requiring an integer amount and an integer index. Perhaps you would like to change to `ERROR_GLOBAL_INDEX_NOT_NUMERIC` for clarity?

Reply:
Good point! Changed in the most recent commit.